### ISSUE TITLE
Solve: True is recognized as ineq and send an error

### ISSUE THREAD

**Issue by latot** [sympy/sympy#11659]
Hi, if we have some True expression in the eqs it can be ignored but actually we get an error:

```
>>> a=solve([true, Eq(x, y)], x, y)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/site-packages/sympy/solvers/solvers.py", line 830, in solve
    return reduce_inequalities(f, symbols=symbols)
  File "/usr/lib64/python2.7/site-packages/sympy/solvers/inequalities.py", line 694, in reduce_inequalities
    rv = _reduce_inequalities(inequalities, symbols)
  File "/usr/lib64/python2.7/site-packages/sympy/solvers/inequalities.py", line 615, in _reduce_inequalities
    symbol of interest'''))
NotImplementedError: 
inequality has more than one symbol of interest
```

Thx. Cya.

**Comment by oscarbenjamin** [comment]
Fixed by #16190 

```
In [3]: solve([true, Eq(x, y)], x, y)
Out[3]: {x: y}
```

**Comment by latot** [comment]
Ty!
